### ISSUE TITLE
sass --watch hangs in 100% CPU after `^C` in 3.3.0

### ISSUE THREAD

**Issue by fhars** [sass/sass#1156]
After finding a workaround for #1151 I am now stuck in `--watch` and have to `kill -9` it from the outside:

```
$ sass --trace --watch test.scss:test.css
>>> Sass is watching for changes. Press Ctrl-C to stop.
>>> New template detected: .#test.scss
>>> Change detected to: test.scss
>>> Deleted template detected: .#test.scss
      write test.css
^C

^C^C

^C^C^C^C
^C^C^C^C^C^C^C
^C^C^C^C^C^C^C^C^C^C^C
^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C
```

Oh, and six minutes later, `top` shows this: 

```
  PID USER      PR  NI  VIRT  RES  SHR S  %CPU %MEM    TIME+  COMMAND 
21772 hars      20   0  816m  26m 3744 S  99,5  0,3   6:00.60 sass
```

**Comment by SamyCookie** [comment]
I confirm this bug.

OS : Debian testing 64bit
version: 3.3.0r c5

**Comment by nex3** [comment]
What do you get if you run `gem list listen`?

**Comment by fhars** [comment]
see #1151

**Comment by SamyCookie** [comment]
@nex3 listen 2.4.1
I have manually downgraded 'listen' because 2.7 version broke sass starting (as describe in #1151).
But now it don't want to exit properly.

**Comment by nex3** [comment]
Other users who have been seeing this: are you also on 2.4.1?

**Comment by nex3** [comment]
This is almost certainly caused by guard/listen#146. Once #1163 is fixed, there should be no need to use listen 2.4.1, so I'm closing this out.

**Comment by SamyCookie** [comment]
After upgrade sass to 3.3.1 and listen to 2.7, all is fine for me. Thanks !
